src/style.css

```diff
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf") format("truetype"),
+    url("fonts/Avenir-Heavy.ttf") format("truetype"),
+    url("fonts/Avenir.ttc") format("ttc");
+  font-style: normal;
+  font-weight: 300;
+}
+
+.input-field {
+  position: relative;
+  margin-top: 20px;
+}
+
+.input-form {
+  width: 100%;
+  font-weight: 300;
+  font-family: "Avenir", Helvetica, sans-serif;
+  box-sizing: border-box;
+  padding-left: 62px;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input-img-big {
+  display: inline-block;
+  position: absolute;
+  background: transparent no-repeat url("images/Search.svg");
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  top: 25px;
+  color: #687c94;
+}
+
+.input-img-small {
+  display: inline-block;
+  position: absolute;
+  background: transparent 80% 80% no-repeat url("images/Search.svg");
+  height: 19px;
+  width: 19px;
+  left: 8px;
+  top: 12px;
+  color: #687c94;
+  transform: scale(0.6);
+}
```

Comment:
why it is still `big` and `small`
look, your code is repeated - don't repeat yourself

Reply:
ok, changed